test/mocha/rest_spec.js

```diff
@@ -124,6 +124,19 @@ describe('Supports form fields', function() {
   });
 });
 
+describe('Supports set and unset param', function() {
+  it('supports set and unset with multiple params of the same name', async function() {
+    const res = await axiosInstance.get('set-unset-params.html?foo=bar&foo=baz'
+    // if URL parameters are supplied via params object, mocha will only send one param 
+    // of a given name, so we must include params in the query string
```

Comment:
See https://stackoverflow.com/questions/42898009/multiple-fields-with-same-key-in-query-params-axios-request. I think the approach in this PR does the trick, but if anyone feels there's a superior method, I could switch to that.